src/main/java/uk/gov/companieshouse/web/pps/models/PenaltyReferenceChoice.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.companieshouse.web.pps.models;
+
+import jakarta.validation.constraints.NotNull;
+
+public class PenaltyReferenceChoice {
+    @NotNull(message = "Select what the penalty reference starts with")
+    private String selectedPenaltyReference;
```

Comment:
Same here, I would expect a unit test. @dgordon-ch are you happy that other tests are hitting this code and that's enough?

Reply:
I would say if the coverage exists upstream it should generally be fine for model/domain classes at least. Happy to cover it it's flagged by Sonar though.

Reply:
Sonar isn't complaining, just me :) That's grand.